### PR TITLE
Impement clerk for auth

### PR DESCRIPTION
**Pull request by kharann**
## Changelog
- Changes main auth to use clerk
- Move Ory functionality to API routes
- The OIDC ory hydra routes uses Clerk to check if the user is logged in
- Fix cross origin authenticated calls

### CODE REVIEW

File: apps/dashboard/src/pages/api/auth/[...nextauth].ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< Updated upstream
```
Comment (matsjla) [inline apps/dashboard/src/pages/api/auth/[...nextauth].ts:1]: :thinking:

File: apps/dashboard/src/app/layout.tsx [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
-
+export default function RootLayout({ children }: PropsWithChildren) {
   return (
     <html lang="en">
       <head />
       <body>
         <MantineColorSchemeProvider>
           <MantineProvider>
-            <AuthProvider session={session}>
+            <AuthProvider session={null}>
```
Comment (matsjla) [inline apps/dashboard/src/app/layout.tsx:16]: What's going on here?

File: apps/dashboard/src/app/event/EventCommitteeDetailsForm.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { FC } from "react"
```
Comment (matsjla) [inline apps/dashboard/src/app/event/EventCommitteeDetailsForm.tsx:1]: This file is outdated and removed in main

File: apps/dashboard/src/app/event/EventDetailsModal.tsx [hunk 4]
```diff
@@ -0,0 +1,103 @@
+import { Card, Grid, Modal, Text, Tabs, Title, useMantineTheme } from "@mantine/core"
```
Comment (matsjla) [inline apps/dashboard/src/app/event/EventDetailsModal.tsx:1]: This file is outdated and removed in main

**Review by matsjla — CHANGES_REQUESTED** [review]

File: apps/dashboard/src/app/layout.tsx (hunk 2 above)
Comment (kharann) [reply apps/dashboard/src/app/layout.tsx:16]: Session is initially null until it's set through nextauth

**Review by matsjla — APPROVED** [review]

**Pull request merged**